src/Components/Navbar/UserNavbar.js

```diff
@@ -111,17 +162,17 @@ export default function UserNavBar(props) {
                 </NavItem>
               </DropdownItem>
               {props.user && props.user.accessLevel >=
-              membershipState.MEMBER && (
+                membershipState.MEMBER && (
```

Comment:
hmm looks like prettier changed a bunch of lines unecesarrily. id move these indented lines back to where they were and double check style with `npm run lint`

Reply:
When I ran `npm run lint ` everything came back okay with the current formatting.

Reply:
if u ever did the mosh tutorial and enabled one of the vscode extentions he asked u to install, u will have to uninstall it bc it messes with the way we structure/align our code

Reply:
i uninstalled it!

Reply:
try saving the file, and quitting out of vscode and opening it up again, if the indents are still weird u may have to manually fix some of the changes this time around